src/components/reportRepair/notEligibleNonEmergency.js

```diff
@@ -0,0 +1,26 @@
+import {
+  H1,
+  GridRow,
+  GridCol,
+  Paragraph
+} from 'govuk-react';
+
+const NotEligibleNonEmergency = () => {
+  return (
+    <GridRow>
+      <GridCol setWidth="two-thirds">
+        <H1>The council may not be responsible for repairs at this property</H1>
```

Comment:
The title doesn't match the [prototype](https://southwark-repairs-alpha-master.herokuapp.com/beta/v2/not-eligible-communal-repairs) - should be "For communal repairs places call us during the office hours below"
```suggestion
        <H1>For communal repairs places call us during the office hours below</H1>
```

The page should list opening hours too